README.md

```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
 and be a talking-point during the interview. A poor analysis will not necessarily
 disqualify you from the job.
 
-### Your Task:
-Use Python to perform a simple analysis of the data table from the following url:
-https://firms.modaps.eosdis.nasa.gov/active_fire/viirs/shapes/zips/VNP14IMGTDL_NRT_Global_7d.zip
-We would like you to analyse the data in relation to the areas of interest in the ares.geojson file.
-At minimum we would like to see at least one map and statistical chart.
+### The data
+Our main focus right now are supply chains, and how they affect the big challenges we face: climate change, water&food security and biodiversity lost. 
 
-### Hints and resources
+We would like you to use one of the datasets we are familiar with: [Mapspam](https://www.mapspam.info/data/)
 
-Add your results to your Github profile in a Jupyter notebook and be prepared to discuss them with us!
-If you don't have a github profile, and you don't know how to get started, we reccomend the following resources:
+### The challenge
+
+We would like you to analyse the data in relation to the areas of interest in the [areas.geojson](./areas.geojson) file and contextualize it.
+
+At minimum we would like to see at least one map and a statistical chart.
 
-https://www.codecademy.com  (a nice git/github tutorial)
-https://software-carpentry.org/lessons/ (Open source lessons for Science-flavoured people)
+### The tools
+Use whatever ones you are comfortable with or you think are the best for the job.
+
+### How should I deliver the results?
+Add your results to your Github profile in a Jupyter notebook and be prepared to discuss them with us!
 
+The technical solution is not the only thing that we value, also your approach and explanations.
 
-Possible libraries you may want to consider: pandas, geopandas, fiona, rasterio, shapely, folium
+### How much time should I spend?
+As little as possible! We understand you probably have other duties so take your time.
 
-Metada for viirs
-https://earthdata.nasa.gov/earth-observation-data/near-real-time/firms/viirs-i-band-active-fire-data
+### What if I have questions?
+Emails us your questions at info@vizzuliaty.com
 
-Extra practica hint:
-If you have trouble setting-up your system, we reccomend using Anaconda Python.
-https://www.anaconda.com/distribution/
```

Comment:
I like to have some tips about were to go if they get stuck, maybe not the ones that were in the previous challenge but some guided information can help

Reply:
No, if they get stuck the can ask. Or demonstrate their independence and use of Google, so I will be against any tip in the challenge.